src/MarginTrading.Brokers/MarginTrading.AccountMarginEventsBroker/Repositories/SqlRepositories/AccountMarginEventsReportsSqlRepository.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Common.Log;
+using Dapper;
+using MarginTrading.AccountMarginEventsBroker.Repositories.Models;
+using MarginTrading.BrokerBase;
+using System;
+using System.Data;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+
+namespace MarginTrading.AccountMarginEventsBroker.Repositories.SqlRepositories
+{
+    internal class AccountMarginEventsReportsSqlRepository : IAccountMarginEventsReportsRepository
+    {
+        private const string TableName = "AccountMarginEventsReports";
+        private const string CreateTableScript = "CREATE TABLE [{0}] (" +
```

Comment:
Duplicated with .sql file. It is better to have only 1 copy of script.

Reply:
SQL file removed